other/materials_designer/specific_examples/defect_creation_island.ipynb

```diff
@@ -267,9 +267,9 @@
    "cell_type": "code",
```

Comment:
<p>Here and in documentation:</p><p> </p><ul><li>defect_surface_island_titanium_nitride</li><li>defect_point_substitution_graphene</li><li>interface_bilayer_twisted_nanoribbons_boron_nitride</li><li>interface_bilayer_twisted_commensurate_lattices_molybdenum_disulfide</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>